jbcli/jbcli/cli/jb.py

```diff
@@ -553,6 +552,55 @@ def prompt_interval():
     stash.put("interval", answer)
 
 
+def prompt_name():
+    question = [
+        {
+            "type": "input",
+            "name": "name",
+            "message": "Enter First and Last Name separted by space",
+        }
+    ]
+    answer = prompt(question)["name"]
+    name_parts = answer.split(' ')
+    stash.put("firstname", name_parts[0])
+    stash.put("lastname", name_parts[1])
+
+
+def prompt_email():
+    question = [
+        {
+            "type": "input",
+            "name": "email",
+            "message": "Enter email address to be created",
+        }
+    ]
+    answer = prompt(question)["email"]
+    stash.put("email", answer)
```

Comment:
I think it would be nice to future-proof this code by putting the user list into the TOML file like this, using the "array of tables" syntax: https://toml.io/en/v1.0.0#array-of-tables

```toml

[[users]]
firstname = "Christopher"
lastname = "Armstrong"
email = "chris.armstrong@juiceanalytics.com"
user_extra = {...}

[[users]]
email = "bob@coolguy.io"
firstname = "..."
lastname = "..."
user_extra = {...}
```

This way we can set up local testing users with different user extras.

I'm pretty sure this will work if you just do `stash.put("users", list_of_dicts)`